forge-gui/res/cardsfolder/upcoming/thief_of_existence.txt

```diff
@@ -0,0 +1,11 @@
+Name:Thief of Existence
+ManaCost:1 C G
+Types:Creature Eldrazi
+PT:3/4
+K:Devoid
+T:Mode$ SpellCast | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When you cast this spell, exile up to one target noncreature, nonland permanent an opponent controls with mana value 4 or less. If you do, CARDNAME gains "When this creature leaves the battlefield, target opponent draws a card."
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Permanent.nonCreature+nonLand+OppCtrl+cmcLE4 | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target noncreature, nonland permanent an opponent controls with mana value 4 or less | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Triggers$ LeavesTrig | Duration$ Permanent | ConditionDefined$ ExiledWith | ConditionPresent$ Card
```

Comment:
this condition check is bad:
consider the case where this triggers twice
- now if you only exile a card with the first (topmost) but don't target with the other you'd still animate it with both!

Reply:
Nice